test/stats.bats

```diff
@@ -24,18 +24,63 @@ function teardown() {
 
     # then
     JSON="$output"
-    echo $JSON | jq -e '.stats[0].attributes.id != ""'
+    run echo $JSON | jq -e '.stats[0].attributes.id != ""'
```

Comment:
As I pointed out in https://github.com/cri-o/cri-o/pull/3933/commits/6886573e612078d3684741d9508778df5b8370de#r463800061, this change is useless.

The more correct change would be to remove `[ "$status" -eq 0 ]` line.